test/test_torch.py

```diff
@@ -891,10 +891,6 @@ def test_horovod_alltoall(self):
         rank = hvd.rank()
         size = hvd.size()
 
-        # This test does not apply if using gloo controller
-        if hvd.gloo_enabled():
```

Comment:
There are several other alltoall tests below this that still contain this skip for gloo. Please remove from the remaining alltoall tests.